.github/workflows/update_rasa_x_version.yml

```
@@ -0,0 +1,95 @@
+name: Update Rasa X version
+
+on:
+  schedule:
+    # Run once a day at 08:00
+    - cron: '0 8 * * *'
+  workflow_dispatch: {}
+
+env:
+  RASABOT_GITHUB_TOKEN: ${{ secrets.RASABOT_GITHUB_TOKEN }}
+  RASABOT_USERNAME: Roberto
+  RASABOT_EMAIL: 43567378+rasabot@users.noreply.github.com
+
+
+jobs:
+  open_pull_request:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Python 3.7 🐍
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Checkout rasa-x-helm github repository 🕝
+        uses: actions/checkout@v2
+        with:
+          # The workflow needs to push to a new branch that changes a workflow file.
+          # To do this, we need the rasabot service account.
+          token: ${{ env.RASABOT_GITHUB_TOKEN }}
+
+
+      - name: Get latest Rasa X minor 🏷
+        run: |
+          DOCKERHUB_TAGS_URL="https://registry.hub.docker.com/v2/repositories/rasa/rasa-x/tags?page_size=10000"
+          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | head -n1 | sed 's/[a-z].*//')
```

We should use only a stable version.
```suggestion
          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | head -n1 | sed 's/[a-z].*//')
```
```suggestion
          LATEST_RASA_X_MINOR=$(curl -s ${DOCKERHUB_TAGS_URL} | jq -r '.results[].name' | grep -vE 'refs|aws|latest|stable|master|ls|list' | sort -Vr | grep -E "^[0-9.]+$" | head -n1')
```